tidb-server/main.go

```diff
@@ -528,6 +530,9 @@ func setGlobalVars() {
 	tikv.RegionCacheTTLSec = int64(cfg.TiKVClient.RegionCacheTTL)
 	domainutil.RepairInfo.SetRepairMode(cfg.RepairMode)
 	domainutil.RepairInfo.SetRepairTableList(cfg.RepairTableList)
+	executor.GlobalDiskUsageTracker = disk.NewGlobalDisTracker(cfg.TempStorageQuota)
+	action := &disk.GlobalPanicOnExceed{}
+	executor.GlobalDiskUsageTracker.SetActionOnExceed(action)
```

Comment:
Perhaps these codes do not belong to `setGlobalVars`. How about using an `init` function in executor package.

Reply:
We could initialize and set panicAction in the init function in executor package, but I' afraid we have to set byteslimit here  (or anywhere else except init function).